config/tech-docs.yml

```diff
@@ -5,22 +5,32 @@ host: reliability-engineering.cloudapps.digital
 show_govuk_logo: false
 service_name: Reliability Engineering
 service_link: /
-phase: Internal
+phase: INTERNAL
 
 # Links to show on right-hand-side of header
 header_links:
   Documentation: /
   GitHub: https://github.com/alphagov/reliability-engineering
 
+# Enables search functionality. This indexes pages only and is not recommended for single-page sites.
 enable_search: true
 
 # Tracking ID from Google Analytics (e.g. UA-XXXX-Y)
 ga_tracking_id:
 
+# Enable multipage navigation in the sidebar
+multipage_nav: true
+
+# Enable collapsible navigation in the sidebar
+collapsible_nav: true
+
 # Table of contents depth – how many levels to include in the table of contents.
 # If your ToC is too long, reduce this number and we'll only show higher-level
 # headings.
-max_toc_heading_level: 6
+max_toc_heading_level: 2
```

Comment:
I know I suggested 2 rather than 3 but would be interested in hearing Emily's thoughts too I think